travel.py

```diff
@@ -9,6 +9,44 @@
 sys.setdefaultencoding("utf-8")
 
 
+# 'nick' : ['dep', 'dest', 'time', 'date']
+searched = {}
```

Comment:
@mrshu No, I think storing data like these ones is not neccessity in this plugin.